blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -150,6 +150,7 @@
     
     // how long we typically wait to schedule a rollup.
     ROLLUP_DELAY_MILLIS("300000"),
+    DELAYED_METRICS_ROLLUP_DELAY_MILLIS("300000"),
```

Comment:
If you set it same as normal, there is no behavior change. Also I dont know what default value we want to set this to? May be 20 mins?